services/ui-src/src/components/pages/Export/McparPdfExport.tsx

```diff
@@ -0,0 +1,126 @@
+import { Box, Heading } from "@chakra-ui/react";
+import { StickyBanner, ReportContext, Table, FieldsSection } from "components";
+
+import { useContext } from "react";
+import { convertDateUtcToEt } from "utils";
+
+import addEditProgram from "../../../forms/addEditProgram/addEditProgram.json";
+
+export const McparPdfExport = () => {
+  const data = useContext(ReportContext);
+  const chipCopy = addEditProgram.fields.filter(
+    (f) => f.id === "combinedData"
+  )[0];
+
+  return (
+    <Box data-testid="mcparPdfExport" sx={sx.container}>
+      <StickyBanner />
+      {data.report && (
+        <Box sx={sx.innerContainer}>
+          <Heading as="h1" sx={sx.heading}>
+            {`Managed Care Program Annual Report (MCPAR) for ${data.report.fieldData.stateName}: ${data.report.programName}`}
+          </Heading>
+
+          <Table
+            shrinkCells
+            sx={sx.metaTable}
+            content={{
+              headRow: ["Due Date", "Last edited", "Edited By", "Status"],
+              bodyRows: [
+                [
+                  `${convertDateUtcToEt(data.report.dueDate)}`,
+                  `${convertDateUtcToEt(data.report.lastAltered)}`,
+                  `${data.report.lastAlteredBy}`,
+                  `${data.report.status}`,
```

Comment:
Suggestion: Don't need to have these as literals -- they can all just be normal like:
`convertDateUtcToEt(data.report.dueDate),`...